java-auto-instrumetation/pkg/controller/javaautoinstrumentation/javaautoinstrumentation_controller.go

```diff
@@ -0,0 +1,390 @@
+package javaautoinstrumentation
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	appv1 "k8s.io/api/apps/v1"
+	"strings"
+	"time"
+
+	javaautoinstrv1alpha1 "github.com/SumoLogic/sumologic-kubernetes-collection/java-auto-instrumentation/pkg/apis/javaautoinstr/v1alpha1"
```

Comment:
This import should be at the bottom of the imports, in separate section.

The non-systems ones from the top should come here

Reply:
👍